### PR TITLE
[WIP] try new protobuf

### PR DESCRIPTION
**Pull request by riteshghorse**
**Please** add a meaningful description for your change here

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

File: sdks/python/apache_beam/utils/proto_utils.py [hunk 1]
```diff
@@ -123,8 +126,25 @@ def pack_Struct(**kwargs):
 def from_micros(cls, micros):
   # type: (Type[TimeMessageT], int) -> TimeMessageT
   result = cls()
-  result.FromMicroseconds(micros)
-  return result
+  if isinstance(result, duration_pb2.Duration):
+    result.FromMicroseconds(micros)
+    return result
+  elif isinstance(result, timestamp_pb2.Timestamp):
+    result.seconds = micros // _MICROS_TO_SECONDS
```
Comment (liferoad) [inline sdks/python/apache_beam/utils/proto_utils.py:133]: can we add some comments here about why we do these conversions?

**Comment by riteshghorse** [comment]
CC: @tvalentyn

File: sdks/python/apache_beam/utils/proto_utils.py [hunk 2]
```diff
@@ -36,6 +35,10 @@
 
 message_types = (message.Message, )
 
+_MICROS_TO_SECONDS = 1000000
```
Comment (tvalentyn) [inline sdks/python/apache_beam/utils/proto_utils.py:38]:
```suggestion
_SECONDS_TO_MICROS = 10**6
```

ditto elsewhere

File: sdks/python/apache_beam/utils/proto_utils.py (hunk 1 above)
Comment (tvalentyn) [reply sdks/python/apache_beam/utils/proto_utils.py:133]: +1. add brief comment why we don't use   timestamp_pb2.Timestamp.FromMicroseconds()
  and a reference an issue that has details.

**Pull request closed without merging**